### ISSUE TITLE
[Bug]: CI fails on `main`: Trigger benchmarks: All jobs have failed - Start CI Run / starter (push) fails 

### ISSUE THREAD

**Issue by mrfh92** [helmholtz-analytics/heat#1549]
### What happened?

Trigger benchmarks workflow fails on main: 

### Error message or erroneous outcome

for "create status":
```shell
Run curl -L -X POST \
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   283  100   103  100   180    546    954 --:--:-- --:--:-- --:--:--  1505
curl: (22) The requested URL returned error: 404
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest",
  "status": "404"
}
Error: Process completed with exit code 22.
```

### Version

main (development branch)

### Python version

None

### PyTorch version

None

**Comment by mrfh92** [comment]
Now we also have: Start CI Run / starter (push) Failing after 11s

**Comment by JuanPedroGHM** [comment]
Fixed with #1583
